Sources/Internals/PassthroughView.swift

```diff
@@ -76,19 +76,21 @@ final class PassthroughView: UIView {
 	}
 
 	private func updatePositioning() {
-		var frame = superview?.bounds ?? .zero
-
-		if let superview = superview as? UIScrollView {
-			if #available(iOS 11.0, *) {
-				frame = frame.inset(by: superview.safeAreaInsets)
-			} else {
-				frame = frame.inset(by: superview.contentInset)
+		DispatchQueue.main.async {
```

Comment:
This ensures that `safeAreaInsets` has the correct value set.

Reply:
I think this was to handle some UIKit craziness on maybe older systems, where the stateful view would be pushed down in some cases. This should be more extensively tested in some apps to ensure we're not breaking a bunch of them. Might be better to create a separte PR for this?

Reply:
Hopefully this won't create endless layout loops 😅